### PR TITLE
[ADD]DEV Recherche avec numéro de séquence

### PR DESCRIPTION
**Pull request by jananjoy**
Integration of  searching functionality for Event, Activity and Task

### CODE REVIEW

**Comment by jananjoy** [comment]
Integration of searching functionality for Event, Activity and Task.

File: project_event/views/project_task_view.xml [hunk 1]
```diff
@@ -40,7 +40,7 @@
                             'default_parent_id': active_id,
                         }" domain="[('activity_task_type','=', 'task')]"/>
                         </page>
-                        <page string="Description">
+                        <page string="DeFscription">
```
Comment (rimbendhaou) [inline project_event/views/project_task_view.xml]: @jananjoy typo error

File: project_event/views/project_task_view.xml [hunk 2]
```diff
@@ -124,32 +136,44 @@
         </field>
     </record>
 
+    <record id="project_task_search" model="ir.ui.view">
```
Comment (rimbendhaou) [inline project_event/views/project_task_view.xml]: @jananjoy activity and task are same (model:project.task), no need to have double search view, you can also inherit from the default search view

File: project_event/views/project_task_view.xml (hunk 1 above)
Comment (jananjoy) [reply project_event/views/project_task_view.xml]: @rimbendhaou   good catch. I changed it

File: project_event/views/project_task_view.xml (hunk 2 above)
Comment (jananjoy) [reply project_event/views/project_task_view.xml]: @rimbendhaou   : As per your comment I have changed with single search view with id  "view_project_task_search" ordering the field as code, name and Partner_id and same comment applicable for Event form also

**Review by rimbendhaou — APPROVED** [review]

**Pull request merged**